pkg/archive/archive_linux_test.go

```diff
@@ -21,7 +21,8 @@ import (
 // ├── d2     # opaque, 0750
 // │   └── f1 # empty file, 0660
 // └── d3     # 0700
-//     └── f1 # whiteout, 0000
+//
```

Comment:
Extra blank line inserted here for some reason.

Reply:
GO fmt is doing this I belive.